bluesky/callbacks/kafka.py

```diff
@@ -0,0 +1,106 @@
+import asyncio
+
+from kafka import KafkaProducer, KafkaConsumer
+from bluesky.run_engine import Dispatcher
+import pickle
+
+from bluesky.callbacks.core import CallbackBase
+
+from ..run_engine import Dispatcher, DocumentNames
+
+import time
+
+
+class Producer(CallbackBase):
```

Comment:
If you are not going to use the simple dispatch method from `CallbackBase` I would not bother inheriting from it.